src/components/itemContextMenu.js

```diff
@@ -116,6 +117,16 @@ import toast from './toast/toast';
             }
         }
 
+        // Multiselect is currrently only ran on long clicks of card components
```

Comment:
I don't believe the UI for the multiselect will work on TVs either. I would suggest adding this inside the check on line 102 and have it be the first item in this group (preceding add to collection and playlist).

Reply:
Thanks! Good catch, I didn't consider tvs.

I made the changes locally and then tried push back into my branch.
I think this was a mistake as its closed the pull request.